doc-content/enterprise-only/installation/install-options-proc.adoc

```diff
@@ -10,7 +10,7 @@ Depending on your environment and project requirements, choose one of the follow
 ====
 
 * Download and run the executable JAR installer for installation on {EAP} 7.1 or {JWS} {JWS_VERSION}. The installer graphical user interface guides you through the installation process.
-* Download one of the following ZIP files. The ZIP files do not require a graphical user interface so you can install {PRODUCT} using SSH.
+* Download one of the following ZIP files.
```

Comment:
Because you removed the second sentence, add a  colon:
Download one of the following ZIP files:

Also, I don't see this text in the rendered docs so check that this file is actually in the assembly. It's possible  that it is obsolete.

Reply:
I think that it is obsolete. I searched and did not find this file in the install assembly or any other assembly. Made the change anyway.